examples/KNNImage/sketch.js

```diff
@@ -32,6 +32,19 @@ function setup() {
     train(2);
   });
 
+  // Reset buttons
+  resetBtnA = select('#resetA');
+  resetBtnA.mousePressed(() => {
```

Comment:
I'll probably adjust this to not have arrow syntax just to be super beginner friendly (arrow syntax a-ok in teachable machines demo though!)